src/server/controllers/users.js

```diff
@@ -150,13 +150,7 @@ module.exports = {
         defaults: userObject})
         .spread( async (createdUser, created) => {
           if(!created) {
-            await User.update(req.body,{
-              where: {
-                email: userObject.email
-              },
-              returning: true
-            });
-            return res.status(200).send({ message: 'the user role has been updated' });
+            return res.status(400).send({ message: 'The user with that email address already exists' });
```

Comment:
@mojo706 Update this functionality to cater for a user who was created thru creating an incident. It means their role ID is still for a regular user. So within check that the role ID matches the role ID on the invite, if not update their role Id and send them the invite email. If they exist and their roleId marches the roleId on the invite, then reply with the above message: `The user with that email address already exists as an {assignee/admin}. Try updating their role`

Reply:
okay